### PR TITLE
chore (refs T34968): minor improvements in DpBulkEditHeader Props.

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://yaits.demos-deutschland.de/T34968
https://github.com/demos-europe/demosplan-ui/pull/615

- minor improvements in DpBulkEditHeader Props.
- edit Trans-Key.

### CODE REVIEW

**Review by salisdemos — APPROVED** [review]

File: translations/messages+intl-icu.de.yml [hunk 1]
```diff
@@ -964,7 +964,11 @@ elements.bulk.edit.change.state.hint: "Planungsdokumente, die bereits den hier g
 elements.bulk.edit.change.state.success: "Die automatische Umstellung wurde bei <strong>{changed}</strong> von <strong>{total}</strong> Planungsdokumenten erfolgreich aktiviert."
 elements.create.subcategory: "Neue Unterkategorie erstellen"
 elements.export: "Planungsdokumente herunterladen"
-elements.selected: "Elemente ausgewählt"
+elements.selected: |
+    {count, plural,
+        =1 { Element ausgewählt }
+        other { Elemente ausgewählt }
```
Comment (spiess-demos) [inline translations/messages+intl-icu.de.yml:970]: We actually have to add the number. See https://github.com/demos-europe/demosplan-core/blob/40ee03ff06d5986bf4568f448c6185aa0204af26/translations/messages+intl-icu.de.yml#L329

```suggestion
        =1 { Ein Element ausgewählt }
        other { # Elemente ausgewählt }
```

We also have to change `items.selected.multi.page` the same way.

File: translations/messages+intl-icu.de.yml [hunk 2]
```diff
@@ -2161,6 +2165,11 @@ news.create: "Neue Mitteilung anlegen"
 news.edit: "Mitteilung bearbeiten"
 news.new: "Neue Mitteilung"
 news.notes: "Aktuelle Mitteilungen"
+news.notes.selected: |
+    {count, plural,
+        =1 { Aktuelle Mitteilung }
+        other { Aktuelle Mitteilungen }
```
Comment (spiess-demos) [inline translations/messages+intl-icu.de.yml:2171]:
```suggestion
        =1 { Eine aktuelle Mitteilung ausgewählt }
        other { # aktuelle Mitteilungen ausgewählt }
```

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: translations/messages+intl-icu.de.yml (hunk 1 above)
Comment (spiess-demos) [reply translations/messages+intl-icu.de.yml]: We also have to change `items.selected.multi.page the same way. Currently the number is not included there.

**Review by spiess-demos — APPROVED** [review]
Lgtm (code review only).

File: translations/messages+intl-icu.de.yml (hunk 1 above)
Comment (ahmad-demos) [reply translations/messages+intl-icu.de.yml]: Done in https://github.com/demos-europe/demosplan-core/pull/2245/commits/1433d78d53d6737467aa6672293018b9c1987f55

**Pull request merged**